configs/securitygroups.yml

```diff
@@ -15,39 +15,30 @@ includes:
         hosts:
           - 0.0.0.0/0
 
-    # administrative hosts have unrestricted access on all ports
-    admin-access:
-        proto: -1
+    jumphost-admin-access:
+        proto: tcp
+        ports: [22, 3389, 5900]
         hosts:
-          - 10.22.75.6/31  # admin1a/b
-          - admin1.private.scl3.mozilla.com
-          - openvpn1.corpdmz.scl3.mozilla.com
-          - openvpn1.stage.corpdmz.scl3.mozilla.com
-          - ssh1.corpdmz.scl3.mozilla.com
-          - ssh1.stage.corpdmz.scl3.mozilla.com
-          - nagios1.private.releng.scl3.mozilla.com  # note: includes tcp/5666
-          - nagios1.private.releng.mdc1.mozilla.com  # note: includes tcp/5666
-          - vportal1a.ops.scl3.mozilla.com
-          - 10.22.240.0/20  # scl3-vpn-net
-          - 10.22.20.0/25  # admin1.scl3-vpn
           - rejh1.srv.releng.scl3.mozilla.com  # 10.26.48.19
           - rejh2.srv.releng.scl3.mozilla.com  # 10.26.48.20
           - rejh1.srv.releng.mdc1.mozilla.com  # 10.49.48.100
           - rejh2.srv.releng.mdc1.mozilla.com  # 10.49.48.101
+          # TODO: REMOVE vpn and IT jumphost direct access
+          - 10.22.240.0/20  # scl3-vpn-net
+          - ssh1.corpdmz.scl3.mozilla.com
 
-    # observium has universal SNMP access
-    observium:
-        proto: udp
-        ports: [161]
-        hosts:
-          - observium2.private.scl3.mozilla.com
-
-    # infra puppetizes hosts by SSHing to them from the master
-    infra-puppetize:
```

Comment:
Is this just obsolete now ?

Reply:
I think infra-puppetize was only used for the IT-puppet-managed nagios. So they're likely still using it, but we don't need it?

Reply:
The nagios instances were deleted because netops is using canary hosts instead. I think they're managing them with ansible, so infra-puppetize not needed.